packages/context/src/types.ts

```diff
@@ -0,0 +1,121 @@
+import type {
+  UI5SemanticModel,
+  UI5Framework,
+} from "@ui5-language-assistant/semantic-model-types";
+import { ConvertedMetadata } from "@sap-ux/vocabularies-types";
+import type { Manifest } from "@sap-ux/project-access";
+
+export const DEFAULT_UI5_FRAMEWORK = "SAPUI5";
+export const DEFAULT_UI5_VERSION = "1.71.49";
+export const UI5_FRAMEWORK_CDN_BASE_URL = {
+  OpenUI5: "https://sdk.openui5.org/",
+  SAPUI5: "https://ui5.sap.com/",
+};
+export const UI5_PROJECT_TYPE = "UI5";
+export const CAP_PROJECT_TYPE = "CAP";
+export enum DirName {
```

Comment:
Perhaps this should be added to the open-ux-tools as this is needed for other extensions as well.

Reply:
yes. I will convey this message and once averrable in `open-ux-tools` we can clean up.